ci/ray_ci/docker_container.py

```diff
@@ -100,11 +65,9 @@ def _get_image_names(self) -> List[str]:
                 # no tag is alias to gpu for ray-ml image
                 platforms.append("")
 
-        alias_images = []
-        ray_repo = f"rayproject/{self.image_type}"
+        tags = []
         for version in versions:
             for platform in platforms:
-                alias = f"{ray_repo}:{version}-{self.python_version}{platform}"
-                alias_images.append(alias)
-
-        return alias_images
+                tag = f"{version}-{self.python_version}{platform}"
```

Comment:
thinking about this, isn't there a case where the python version is also omitted?

like `rayproject/ray:2.7.0` is actually valid tag? is that lost now?

Reply:
ah, you're right, I missed that part in the documentation

`The optional Python version tag specifies the Python version in the image. All Python versions supported by Ray are available, e.g. py37, py38, py39 and py310. If unspecified, the tag points to an image using Python 3.7.`

Reply:
Fixed in https://github.com/ray-project/ray/pull/39917